corehq/apps/translations/views.py

```diff
@@ -225,3 +238,119 @@ def post(self, request, *args, **kwargs):
                 except ResourceMissing:
                     messages.add_message(request, messages.ERROR, 'Resource not found')
         return self.get(request, *args, **kwargs)
+
+
+@location_safe
+@method_decorator([toggles.APP_TRANSLATIONS_WITH_TRANSIFEX.required_decorator()], name='dispatch')
+class AppTranslations(BaseTranslationsView):
+    page_title = ugettext_lazy('App Translations')
+    urlname = 'app_translations'
+    template_name = 'app_translations.html'
+    section_name = ugettext_lazy("Translations")
+
+    @use_select2_v4
```

Comment:
could also be a class level decorator like the toggle one and then remove the `dispatch` method.

Reply:
I believe `use_select2_v4` would need to be modified for this to work. Not sure what that change should be.

Reply:
okay, thanks @orangejenny